README.md

```diff
@@ -204,6 +204,7 @@ Optionally, you may supply an `acl_api_token`.  This will allow you to create
 ACLs if the anonymous token doesn't permit ACL changes (which is likely).
 The api token may be the master token, another management token, or any
 client token with sufficient privileges.
+### NOTE. If you're already using ACL tokens you should consider not to use HCL due to library using JSON.parse function to create/update tokens
```

Comment:
It's related to those, who faced the same problem as we did. It doesn't work with beforehand set up ACL rules in Hashicorp DSL. And yes, JSON from your Ruby code.